apps/dashboard/app/models/files.rb

```diff
@@ -134,7 +134,7 @@ def self.mime_type_for_preview(type)
     #
     case type
     when 'text/javascript', 'text/css', 'text/yaml', 'text/yml'
-      'text/plain'
+      'text/plain; charset=utf-8'
     else
       type
     end
```

Comment:
I wonder if this can just be text content type?  In any case, in this case I was looking at a shell script that was `text/plain`, which would need to be added to this when block. But what about `text/json`? What about `text/xml` and on and on and on. 

Is there some reason we can't just cast all text formats to `text/plain`?

```ruby
    if %r{text/.*}.match(type).nil?
      type
    else
      'text/plain; charset=utf-8'
    end
```